consensus/polybft/consensus_runtime.go

```diff
@@ -67,6 +67,17 @@ type epochMetadata struct {
 	Commitment *Commitment
 }
 
+type guardedDataDTO struct {
```

Comment:
can we have better naming for this, it seems like a generic name?

Reply:
It has already been  merged, we can change it via another PR.

Reply:
> can we have better naming for this, it seems like a generic name?
:+1: